pkg/receive/multitsdb.go

```diff
@@ -259,15 +261,15 @@ func (t *MultiTSDB) getOrLoadTenant(tenantID string, blockingStart bool) (*tenan
 			)
 		}
 
-		s := NewFlushableStorage(
+		s, err := tsdb.Open(
 			dataDir,
 			logger,
 			reg,
```

Comment:
same here, the FlushableStorage implementation ensured that a each opened tsdb could re-register metrics every time that a database reload was triggered. Doing two `tsdb.Open`s should cause two calls withing the `tsdb` package to `MustRegister`, which will cause a panic, no?